aea/protocols/fipa/message.py

```diff
@@ -43,8 +43,8 @@ class Performative(Enum):
         MATCH_ACCEPT = "match_accept"
         DECLINE = "decline"
         INFORM = "inform"
-        ACCEPT_W_ADDRESS = "accept_w_address"
-        MATCH_ACCEPT_W_ADDRESS = "match_accept_w_address"
+        ACCEPT_W_INFORM = "accept_w_inform"
+        MATCH_ACCEPT_W_INFORM = "match_accept_w_inform"
```

Comment:
Why an `ACCEPT` or `MATCH_ACCEPT` message can't have an optional `inform` instead of creating a new performative for that?

Reply:
@marcofavorito it can but it makes the definition of `ACCEPT` less tight. I think it is good to avoid optional fields were possible with performatives as it in practice means more conditional checks. I would prefer to even get rid of the `info` dict and have it fully specified by performative, but that would be excessive.